mne_qt_browser/_pg_figure.py

```diff
@@ -3503,6 +3509,19 @@ def __init__(self, **kwargs):
         ahelp.triggered.connect(self._toggle_help_fig)
         self.mne.toolbar.addAction(ahelp)
 
+        # 2nd Toolbar accommodating the Text Boxes of Scalings
+        self.addToolBarBreak()
+        self.mne.toolbar2 = self.addToolBar("Scales")
+        # Scalings Text Boxes
+        self.scale_boxes = OrderedDict()
+        for chtype in [ct for ct in self.mne.ch_types_ordered if ct != "stim"]:
+            lbl = QLabel(chtype.upper())
```

Comment:
Copying @larsoner's suggestion here: 

> For the labels, though, instead of using ch_type.upper() or whatever it is now, you should use DEFAULTS["titles"].get(ch_type, ch_type.upper()) or similar to get nice human readable ones like we do here